api/linux_handler.go

```diff
@@ -20,3 +22,12 @@ func FreeLinuxHandler(w http.ResponseWriter, r *http.Request) {
 	}
 	json.NewEncoder(w).Encode(response)
 }
+
+func DfLinuxHandler(w http.ResponseWriter, r *http.Request) {
+	response, err := df.Df()
+	if err != nil {
+		http.Error(w, "Failed to retrieve disk info: "+err.Error(), http.StatusInternalServerError)
+		return
+	}
+	json.NewEncoder(w).Encode(response)
+}
```

Comment:
Could you also add this part to other handlers and commands? It seems that the error handling is not implemented for other handlers as well.

Reply:
I'm guessing you are talking about UptimeLinuxHandler. 
IMO I think this PR should only contain the changes of `df` command.
So creating a new issue for this could be the good idea.

Reply:
k